content/src/main/content/jcr_root/apps/acs-commons/components/authoring/component-cloner/component-cloner.html

```diff
@@ -0,0 +1,32 @@
+<!--/*
+  ~  ACS AEM Commons Package
+  ~
+  ~ Copyright 2018 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+ */-->
+
+<!--/*including the classic listener javascript logic during author mode*/-->
+<sly data-sly-test="${!wcmInit.wcmmode.disabled}"
+     data-sly-use.clientLib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-call="${clientLib.js @ categories='acs-commons.component.cloner'}">
```

Comment:
Does this include the classic UI clientlib on the Touch UI?

Reply:
This call is to include the classic UI clientlib. The Touch-UI clientlib is included with the OOTB authoring via 'cq.authoring.dialog' category.

Reply:
Right - my concern is whether the classic UI clientlib is being loaded in the Touch UI (it should, if it can be prevented)